NVIDIA A100-SXM4-40GB 7
{'dataset_name': 'ogbg-ppa', 'basis': 'GSO', 'epsilon': [0], 'power': [1], 'seeds': [0, 1, 2, 3, 4], 'num_workers': 4, 'hyperparams': {'batch_size': 32, 'epochs': 30, 'learning_rate': 0.0005, 'decay_rate': 0.7, 'milestones': [20, 40, 60], 'weight_decay': 0.0005, 'warmup_epochs': 5}, 'architecture': {'layers': 4, 'hidden': 256, 'pooling': 'S', 'dropout': 0.5}, 'commit_id': '6fa64202477020bcba460f933726586ec95785ca', 'time_stamp': '1672033842', 'directory': '../../../results_spectral_rotation/A100/ogbg-ppa/'}
{'dataset_name': 'ogbg-ppa', 'basis': 'GSO', 'epsilon': [0], 'power': [1], 'seeds': [0, 1, 2, 3, 4], 'num_workers': 4, 'hyperparams': {'batch_size': 32, 'epochs': 30, 'learning_rate': 0.0005, 'decay_rate': 0.7, 'milestones': [20, 40, 60], 'weight_decay': 0.0005, 'warmup_epochs': 5}, 'architecture': {'layers': 4, 'hidden': 256, 'pooling': 'S', 'dropout': 0.5}, 'commit_id': '6fa64202477020bcba460f933726586ec95785ca', 'time_stamp': 1672033846, 'directory': '../../../results_spectral_rotation/A100/ogbg-ppa/', 'seed': 0}
Loading the cached file... (NOTE: delete it if you change the preprocessing settings)
Bases total: 2
{'train': tensor([    0,     1,     2,  ..., 78197, 78198, 78199]), 'valid': tensor([ 78200,  78201,  78202,  ..., 123297, 123298, 123299]), 'test': tensor([123300, 123301, 123302,  ..., 158097, 158098, 158099])}
Traceback (most recent call last):
  File "/home/mingqi/workspace/spectral_rotation/ogbg_a/graphml/ogbg/ppa/./main.py", line 325, in <module>
    main()
  File "/home/mingqi/workspace/spectral_rotation/ogbg_a/graphml/ogbg/ppa/./main.py", line 132, in main
    epoch_idx, train_curve, valid_curve, test_curve, trainL_curve, task_type = run_with_given_seed(config, ts_fk_algo_hp)
  File "/home/mingqi/workspace/spectral_rotation/ogbg_a/graphml/ogbg/ppa/./main.py", line 200, in run_with_given_seed
    train_loader = DataLoader(dataset[1, 2, 3, 8000], batch_size=config.hyperparams.batch_size, shuffle=True,
  File "/home/mingqi/anaconda3/envs/pytorch_ogbg/lib/python3.9/site-packages/ogb/graphproppred/dataset_dgl.py", line 195, in __getitem__
    raise IndexError(
IndexError: Only integers and long are valid indices (got tuple).
